apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -622,16 +638,16 @@ export function WriteInsAdjudicationScreen(): JSX.Element {
           </Button>
         </AdjudicationNav>
       </AdjudicationHeader>
-      <Main flexRow data-testid={`transcribe:${currentWriteIn.id}`}>
+      <Main flexRow data-testid={`transcribe:${currentWriteInId}`}>
         <BallotViews>
-          {writeInDetailView && (
+          {writeInImageView ? (
             <BallotImageViewer
-              key={currentWriteIn.id} // Reset zoom state for each write-in
-              imageUrl={writeInDetailView.imageUrl}
-              ballotBounds={writeInDetailView.ballotCoordinates}
-              writeInBounds={writeInDetailView.writeInCoordinates}
+              key={currentWriteInId} // Reset zoom state for each write-in
+              imageUrl={writeInImageView.imageUrl}
+              ballotBounds={writeInImageView.ballotCoordinates}
+              writeInBounds={writeInImageView.writeInCoordinates}
             />
-          )}
+          ) : null}
```

Comment:
If image isn't ready, just show dark background of ballot viewer. I felt this looks better than "Loading..."